apps/ledger-live-mobile/src/components/BulletList.tsx

```diff
@@ -111,12 +118,14 @@ export function BulletSmallDot() {
 }
 
 class BulletList extends PureComponent<{
-  list: any;
+  list: React.ComponentProps<typeof BulletItem>["value"][];
```

Comment:
wouldn't it be more redable to have

```ts
interface BulletListProps {
  value: React.ReactNode | (() => React.ReactNode);
  index: number;
  animated?: boolean;
  itemContainerStyle?: StyleProp<ViewStyle>;
  itemStyle?: StyleProp<ViewStyle>;
  itemTextStyle?: StyleProp<TextStyle>;
  Bullet: React.ComponentType<unknown>;
}
```

Then use

```ts
export class BulletItem extends PureComponent<BulletItemProps>
```

and
```ts
list: BulletItemProps["value"]
```

Reply:
Does not matter much